packages/katello/katello/katello-debug.sh

```diff
@@ -69,12 +69,6 @@ add_files /var/log/httpd/pulp-http{s,}_access_ssl.log*
 add_files /var/log/httpd/pulp-http{s,}_error_ssl.log*
 add_files /etc/default/pulp*
 
-# MongoDB (*)
```

Comment:
Shouldn't we add the mongo log files? If they move, then we need the new location. It might also be good to add both, even if they're old. I can imagine it could help when there are issues during the migration.

Reply:
switching to syslog, so not needed anymore

Reply:
Would it still make sense to continue sending the the old log files for the migration case? Could be a case of better to have them & not need them than to not have them & need them.